package.json

```diff
@@ -1,19 +1,17 @@
 {
   "name": "@govtechsg/sgds",
-  "version": "2.0.0-alpha.3",
+  "version": "2.0.0-alpha.4",
   "description": "Design System unites Singapore Government digital services around a common visual language and user experience",
   "repository": {
     "type": "git",
     "url": "https://github.com/govtechsg/sgds.git"
   },
   "license": "MIT",
   "author": "Tey Liang Tai",
-  "main": "sgds/sgds.css",
+  "main": "css/sgds.css",
   "files": [
-    "sgds/sass",
-    "sgds/sgds.css",
-    "sgds/sgds.scss",
-    "sgds/sgds.css.map"
+    "sass",
+    "css"
   ],
   "scripts": {
     "build": "npm run packagedownloads && npm run build:site && bundle exec jekyll build",
```

Comment:
Edit build:lib script on line 18

`rm -rf ./css && webpack --mode=production`

Reply:
updated in latest commit [e1de0fe](https://github.com/GovTechSG/sgds/pull/324/commits/e1de0fea65f824acd51c135108c354370cca8ab2)